ALUM/ALUM/Views/LoginPage/LoginPageViewModel.swift

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
note - We might have to move this ViewModel to the ViewModels directory and also add a model in future. No action for now tho